DevProxy.Abstractions/Utils/ProxyUtils.cs

```diff
@@ -158,7 +157,7 @@ public static IList<MockResponseHeader> BuildGraphResponseHeaders(Request reques
                 new ("Date", requestDate),
                 new ("Content-Type", "application/json")
             };
-        if (request.Headers.FirstOrDefault((h) => h.Name.Equals("Origin", StringComparison.OrdinalIgnoreCase)) is not null)
+        if (request.Headers.Contains("Origin"))

```

Comment:
Is this comparison case-insensitive?

Reply:
According to the http specs the headers should be case insensitive, but they come through as they come through. Kestrel parses them to an actual object no matter the casing. So I thought let's make it case insensitive here as well.